src/02-data-types.js

```diff
@@ -95,7 +109,13 @@ function checkIfCharacterIsInString(text, character) {}
  *  isEven(11);
  *  //> false
  */
-function isEven(num) {}
+function isEven(num) { 
+  if (num%2 === 0)
```

Comment:
Lets make sure that our ` if ` statements have a block after them ` if(thisIsTrue) { } `.  We can do an if statement on one line but if we move to multiple lines we need to add a block.  Also - do we need to put an else statement here or can we simply ` return false ` in the event that we don't return true?